frontend/test/tickets/card/ticketsCard.store.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { TicketsCardActions } from "@/v5/store/tickets/card/ticketsCard.redux";
```

Comment:
Not sure whether this is going to be a problem, but I keep seeing these 2 errors printed in the console

![image](https://github.com/3drepo/3drepo.io/assets/43547703/0b3c2216-c7ce-4a2c-a1dc-631cf085e76b)